src/components/Library/EndedVoting.tsx

```diff
@@ -73,6 +68,14 @@ const StVotingWrapper = styled.section`
   flex-direction: column;
 
   position: relative;
+
+  :first-child {
+    margin-left: 2rem;
+  }
+  :last-child {
+    margin-left: 0;
+    margin-right: 2rem;
+  }
```

Comment:
요렇게 해버리면 라이브러리에 항목이 1개밖에 없을 때 `:last-child`가 적용이 되어서, margin-left가 0이 된다 ㅜㅜ! 
나도 궁금해서 수정해봤는데 해결한거같아서 suggestion으로 달아볼게!! 밑에 `MonthVoting.tsx`에도 하나 더 있엉!!!
<img width="537" alt="스크린샷 2023-02-10 오후 10 39 29" src="https://user-images.githubusercontent.com/73213437/218105961-9e3ec04b-cd31-4f04-9977-a8f353193a2d.png">


```suggestion
  margin-right: 1.521rem;

  :last-child {
    margin-right: 2rem;
  }
```

Reply:
데이터 하나일 때를 테스트를 못해봤었구나 내가!! 쭈아 반영해떠! 고마어 현지 똑똑이~